source/css/_common/components/post/post-type.styl

```diff
@@ -1,14 +0,0 @@
-// TODO: Refactor.
-
-.page-home, .page-post-detail {
-  .post-type-quote {
```

Comment:
@1v9 you can help confirm if `.post-type-quote` still exist

Reply:
@stevenjoezhang OK